docs/gallery/index.md

```diff
@@ -0,0 +1,29 @@
+# Gallery of Previous Hackweeks
+
+````{panels}
+:container: full-width
+:column: text-center col-6 col-lg-4
+:card: +my-2
+
+**SnowEx Hackweek**
+^^^
+<img src="https://snow.nasa.gov/sites/default/files/snowex-logo.png" alt="SnowEx logo" width="200" height="150">
++++
+*[Click here](SnowExhackweek) for details*
+
```

Comment:
looks like you can make the entire panel card a clickable link https://jupyterbook.org/content/content-blocks.html?highlight=panels#panels 

```
---

```{link-button} content/panels
:text: Clickable right panel
:type: ref
:classes: stretched-link
```
```